packages/communication-react/CHANGELOG.md

```diff
@@ -4,10 +4,10 @@ This log was last generated on Mon, 24 Jan 2022 23:18:53 GMT and should not be m
 
 <!-- Start content -->
 
-## [1.0.1](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.0.1)
+## [1.0.1-beta.2](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.0.1-beta.2)
```

Comment:
I think this should be `-beta.1`

Reply:
Similarly the compare link bwelow should be 1.0.0...1.0.1-beta.1

Reply:
Err... why? We're releasing `1.0.1-beta.2` next, eh?

Reply:
Oh this is into the release branch not main! my bad ignore me ^_^